tests/test_wave_01.py

```diff
@@ -122,8 +121,10 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][0]["title"] is MOVIE_TITLE_1
+    assert updated_data["watched"][0]["genre"] is GENRE_1
+    assert updated_data["watched"][0]["rating"] is RATING_1
```

Comment:
Nice checking all the relevant keys of the movie! When we have to use a value multiple times that's the result of performing operations (like a calculation or nested data access), it can often be worth the space tradeoff to store it in a variable. This is both to save us the operations of accessing the data each time, and increasing readability.
```suggestion
    watched_movie = updated_data["watched"][0]
    assert watched_movie["title"] is MOVIE_TITLE_1
    assert watched_movie["genre"] is GENRE_1
    assert watched_movie["rating"] is RATING_1
```